src/component.js

```diff
@@ -205,14 +205,24 @@ export default class Component {
       this.model = model;
       this.render();
       this.delegateEvents();
+      if (this.iframe) {
+        this.iframe.el.onload = () => {
+          this.iframe.el.onload = null;
+          this.init();
+        };
+      }
       this._userEvent('afterInit');
       return this;
     })
     .catch((err) => {
       if (err.message.indexOf('Not Found') > -1) {
         logNotFound(this);
       } else {
-        throw err;
+
+        /* eslint-disable no-console */
```

Comment:
If you use `//eslint-disable-next-line no-console`, then you don't have to re-enable it later!